Readability-readerable.js

```diff
@@ -89,7 +106,7 @@ function isProbablyReaderable(doc, options = {}) {
       return false;
     }
 
-    var textContentLength = node.textContent.trim().length;
+    var textContentLength = node.textContent.trim().replace(/\s+/g, " ").length;
```

Comment:
What's the point of the added `replace` here?

Reply:
to count contiguous whitespace as one character, to not inflate text length of sparse content